src/test/java/wooteco/subway/AuthAcceptanceTest.java

```diff
@@ -49,36 +22,15 @@ void myInfoWithBearerAuth() {
         assertThat(memberResponse.getName()).isEqualTo(TEST_USER_NAME);
     }
 
-    public MemberResponse myInfoWithBasicAuth(String email, String password) {
-        // TODO: basic auth를 활용하여 /me/basic 요청하여 내 정보 조회
-        return null;
-    }
-
-    public MemberResponse myInfoWithSession(String email, String password) {
-        // TODO: form auth를 활용하여 /me/session 요청하여 내 정보 조회
-        return null;
-    }
-
+    // @formatter:off
     public MemberResponse myInfoWithBearerAuth(TokenResponse tokenResponse) {
-        // TODO: oauth2 auth(bearer)를 활용하여 /me/bearer 요청하여 내 정보 조회
-        return null;
-    }
-
-    public TokenResponse login(String email, String password) {
-        Map<String, String> params = new HashMap<>();
-        params.put("email", email);
-        params.put("password", password);
-
-        return
-                given().
-                        body(params).
-                        contentType(MediaType.APPLICATION_JSON_VALUE).
-                        accept(MediaType.APPLICATION_JSON_VALUE).
-                when().
-                        post("/oauth/token").
-                then().
-                        log().all().
-                        statusCode(HttpStatus.OK.value()).
-                        extract().as(TokenResponse.class);
+        return given().header("Authorization",
```

Comment:
```suggestion
        return given().header(AuthorizationExtractor.AUTHORIZATION,
```

최대한 의미있는 상수를 활용해주세요!